src/App.tsx

```diff
@@ -1,60 +1,133 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
[CODE STYLE] - Don't use import keyword in non-root components for data that could be passed down from the root component. To follow the checklist item 5, consider preparing the data in the App component and pass it to child components as props.